spec/models/user_spec.rb

```diff
@@ -664,5 +664,11 @@ def user_from_authorization_service(service_name, signed_in_resource, cta_varian
       user.destroy
       expect(user.persisted?).to be false
     end
+
+    it "destroys associated organization memberships" do
+      organization_membership = create(:organization_membership, user_id: user.id, organization_id: org.id)
+      user.destroy
+      expect { organization_membership.reload }.to raise_error ActiveRecord::RecordNotFound
```

Comment:
I had to use this work around instead of `organization_membership.persisted?` since `.persisted?` kept returning `true` for some reason.